tests/hikari/test_event_stream.py

```diff
@@ -221,16 +219,10 @@ def test___del___for_active_stream(self):
         streamer._event_type = events.Event
         streamer._active = True
 
-        with mock.patch.object(asyncio, "ensure_future", side_effect=RuntimeError):
-            with unittest.TestCase().assertLogs("hikari", level=logging.WARNING) as logging_watcher:
```

Comment:
Testing logging is not necessary

Reply:
although isn't logging a part of the feature here since this is just a healper feature for alerting ppl using bad pratices on cpython that isn't guarnteed to work well if at all?